meson_options.txt

```diff
@@ -55,8 +55,6 @@ option('initrd', type : 'boolean',
        description : 'install services for use when running systemd in initrd')
 option('compat-mutable-uid-boundaries', type : 'boolean', value : false,
        description : 'look at uid boundaries in /etc/login.defs for compatibility')
-option('nscd', type : 'boolean',
```

Comment:
Need to be removed from meson.build too?

Reply:
Yeah, I got distracted halfway.